### ISSUE TITLE
False positive in "newline-after-var" when using let under another let

### ISSUE THREAD

**Issue by DAvidCoolAutomation** [eslint/eslint#11495]
**Tell us about your environment**

* **ESLint Version:** 5.15.1
* **Node Version:** 10.15.2
* **npm Version:** 6.4.1

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "parser": "babel-eslint",
  "parserOptions": { "ecmaVersion": 6 },
  "env": { "es6": true },
  "rules": {
    "newline-after-var": [1, "always"]
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const x = 1;
const y = 2;

var a = 3;
var b = 4;

let m = 5;
let n = 6;

console.log(x + y);
```

```bash
eslint "**/*.{js,jsx}"
```

**What did you expect to happen?**
No warnings.

**What actually happened? Please include the actual, raw output from ESLint.**
  7:1  warning  Expected blank line after variable declarations  newline-after-var

✖ 1 problem (0 errors, 1 warning)
  0 errors and 1 warning potentially fixable with the `--fix` option.

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by g-plane** [comment]
Hi @DAvidCoolAutomation , thanks for this issue.

The rule "newline-after-var" has been deprecated. We recommend to use [`padding-line-between-statements`](https://eslint.org/docs/rules/padding-line-between-statements).
